settings/template.ini

```diff
@@ -72,6 +72,13 @@ Methods[]=ftps
 Methods[]=webdav
 Methods[]=webdavs
 
+# Add support for text which is started from www. without protocol.
+# For example change text "www.ez.no" to link tags.
+LinkWwwUrlsWithoutProtocol=disabled
+
+HideProtocols[]
```

Comment:
This setting block requires a detailed comment explaining it's use. Here is a suggested example:
```
# Strip protocols from link text when using autolink template operator
# For example link text string "http://www.ez.no". Disabled by default to retain BC.
# Add link protocols to enable this feature.```